src/screens/OtherProfile/index.js

```diff
@@ -606,12 +612,6 @@ const OtherProfile = () => {
     });
   };
 
-  const __handleOnEndReached = () => {
-    if (!isLastPage) {
-      getOtherFeeds(postOffset);
-    }
-  };
-
   const handleRefresh = () => {
     setIsLastPage(false);
     getOtherFeeds(0);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `__handleOnEndReached` function has been removed and replaced with a direct call to `getOtherFeeds(0)` in `handleRefresh`. This seems to be part of the optimization effort to fetch new feeds based on scroll momentum rather than reaching the end of the list. Ensure that this change doesn't affect the user experience negatively by causing unnecessary data fetching or UI updates.

```diff
-  const __handleOnEndReached = () => {
-    if (!isLastPage) {
-      getOtherFeeds(postOffset);
-    }
-  };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->